src/display/font_loader.js

```diff
@@ -66,10 +66,12 @@ class FontLoader {
   }
 
   clear() {
-    for (const nativeFontFace of this.nativeFontFaces) {
```

Comment:
Since we need the original font to create a text layer, we need to preserve the font face here. 
This may increase memory usage but can improve rendering speed when resizing, as we don’t need to load the font again.